geopandas/tests/test_extension_array.py

```diff
@@ -402,17 +402,31 @@ def test_fillna_series(self, data_missing):
         # `geopandas\tests\test_pandas_methods.py::test_fillna_scalar`
         # and `geopandas\tests\test_pandas_methods.py::test_fillna_series`.
 
-    @pytest.mark.skip("fillna method not supported")
-    def test_fillna_limit_pad(self, data_missing):
-        pass
+    def test_fillna_limit_ffill(self, data_missing):
```

Comment:
Hey @Zeroto521, this is looking good to me, and I think these are good tests. However I hope @jorisvandenbossche can weigh in, as I understand it these tests are inherited from pandas, and are set up such that they should be "automatic", via the `data_missing` fixture. So I think potentially we could move these tests to `test_pandas_methods` and re-enable the default tests here (by removing the methods completely). We could also leave the tests here and just make clear with a comment those which are not inherited tests - I also think there's some sense in keeping similar tests grouped closely.

(Perhaps we also should have done the same in https://github.com/geopandas/geopandas/pull/2542, you extented the original overidden test, when we probably could have just used the pandas one)